webapp/views.py

```diff
@@ -1194,6 +1194,38 @@ def dispatch_request(self):
 
 # Cube
 # ===
-def cube_home():
-    data = {"user": "Beth", "badges": ["Badge1", "Badge2"]}
-    return flask.render_template("cube/index.html", **data)
+def cube_microcerts():
+    data = {
+        "user": {"name": "Beth Collins"},
+        "modules": [
+            {
+                "badge": "badge1",
+                "name": "module1",
+                "topics": ["topic1", "topic2", "topic7", "topic8"],
+                "testurl": "http://module1.url",
+                "trainingurl": "http://module3.url",
+                "status": "open",
+                "action": "Test",
+            },
+            {
+                "badge": "badge2",
+                "name": "module2",
+                "topics": ["topic3", "topic4"],
+                "testurl": "http://module2.url",
+                "trainingurl": "http://module3.url",
+                "status": "Failed once",
+                "action": "Retest",
+            },
+            {
+                "badge": "badge3",
+                "name": "module3",
+                "topics": ["topic5", "topic6", "topic9"],
+                "testurl": "http://module3.url",
```

Comment:
In python variable names are usually snake cased. So `testurl` would become `test_url`.
Same thing for `trainingurl`.

Reply:
ahh thanks, I'll fix this now